cirq/__init__.py

```diff
@@ -197,6 +197,7 @@
 from cirq.optimizers import (
     ConvertToCzAndSingleGates,
     DropEmptyMoments,
+    DropIdentities,
```

Comment:
I think this should just be part of `DropNegligible`. Give `IdentityGate` a `def _trace_distance_bound_(self): return 0` and it should work.